css/styles.css

```diff
@@ -341,8 +427,13 @@ ul{
   background-color:#a0a7a4;
   color:#056Ba5 ;
 }
+#top-alert {
+  position: fixed;
+  top: 0;
+  z-index: 99999;
+}
 
-#formbutton:hover {
+.formbutton:hover {
```

Comment:
Only use "#" aka ids for a single element. "#" should apply to only 1 thing and never should be used for multiple elements. 

Good idea to change formbutton to a class since we probably will reuse it.

Reply:
Got it, thank you!